### ISSUE TITLE
Individual app lock 

### ISSUE THREAD

**Issue by Redteamer0101** [akaMrNagar/Mindful#19]
Hi, thank you so much for creating this amazing app.im using this app from long time and my sister started using it recently. But she is facing one issue with the app is lock she wants me to setup lock but it uses system app locker that she can also open. Can you add feature like entire phone will be lock for some time only some assigned apps allowed like phone app to call and some emergency 2-3 apps. Also instead using phone lock create mindful's individual lock and import if I want to lock het phone for 1-2 hr or 30 minites I can remotely if possible

**Comment by akaMrNagar** [comment]
I don't think so, adding remote capabilities requires internet which i don't think is best for Mindful. But we can add our own locking functionality though. But it is going to take time

**Comment by Redteamer0101** [comment]
cool no worries just one more thing if it is fast to do is can we hide it from home screen so I will install app and do confing and only I knows the diapad lock to open app. so she don't know that where to open app for settings.

**Comment by akaMrNagar** [comment]
This is not possible on normal devices it will need root privileges.
